apps/sql-server/fdw/fdw/table.py

```diff
@@ -125,28 +125,30 @@ def connection(class_name: Optional[str],
     src_command = dst_command = connection_command = None
     return_fn = None
 
-    if src_constraint:  # cases 2, 4, 5, or 6
+    if src_constraint or is_system_class:  # cases 2, 4, 5, or 6
         src_command_name = "FindEntity" if src_class[
-            0] != "_" else f"Find[{src_class[1:]}]"
+            0] != "_" else f"Find{src_class[1:]}"
```

Comment:
This line feed is odd. (nothing to doo with the PR though)

Reply:
Yeah, I just let my editor do what it wants with line feeds.